dev-tools/mage/settings.go

```diff
@@ -540,6 +543,7 @@ func parseDocBranch(data []byte) (string, error) {
 // ProjectRepoInfo contains information about the project's repo.
 type ProjectRepoInfo struct {
 	RootImportPath string // Import path at the project root.
+	MountPoint     string // Mount point for the crossBuilder.
```

Comment:
Rather than relating it to crossBuilder I think it's better to explain what it contains. IIUC this is actually the pre-modules canonical root import path. Maybe we should just change the name altogether.

```suggestion
	CanonicalRootImportPath     string // Pre-modules canonical root import path (does not contain semantic import version identifier).
```

If you change to that name then its values need to be `filepath.ToSlash`'ed.